src/ui/components/Overlay/styles.scss

```diff
@@ -13,6 +13,12 @@
   position: fixed;
   top: 0;
   width: 100%;
+  z-index: 2;
+
+  + form {
```

Comment:
That looks like it won't fix everything. It works only because there is a form in the other component, no? Also the use of `+` is usually the last option in CSS.